docs/operational_guide/placement_configuration.md

```diff
@@ -126,16 +126,31 @@ After sending the add command you will need to wait for the M3DB cluster to reac
 
 #### Removing a Node
 
-Send a DELETE request to the `/api/v1/placement/<NODE_ID>` endpoint.
+Send a DELETE request to the `/api/v1/services/m3db/placement/<NODE_ID>` endpoint.
 
 ```bash
-curl -X DELETE <M3_COORDINATOR_HOST_NAME>:<M3_COORDINATOR_PORT(default 7201)>/api/v1/placement/<NODE_ID>
+curl -X DELETE <M3_COORDINATOR_HOST_NAME>:<M3_COORDINATOR_PORT(default 7201)>/api/v1/services/m3db/placement/<NODE_ID>
 ```
 
 After sending the delete command you will need to wait for the M3DB cluster to reach the new desired state. You'll know that this has been achieved when the placement shows that all shards for all hosts are in the `Available` state.
 
 #### Replacing a Node
 
-Currently, the best way to replace a node (due to hardware failure or any other reason) is to perform a node remove followed by a node add. We will eventually support node replacement as a single operation, but that is currently not implemented.
-
+Send a POST request to the `/api/v1/services/m3db/placement/replace` endpoint containing hosts to replace and candidates to replace it with.
 
+```bash
+curl -X POST <M3_COORDINATOR_HOST_NAME>:<M3_COORDINATOR_PORT(default 7201)>/api/v1/services/m3db/placement/replace -d '{
```

Comment:
Does this all work out correctly with various numbers of inputs and outputs? I.E if I want to replace one node with two, or remove two and add one etc

Reply:
It should so long as that algorithm considers that change valid, since we don't do much rather than be a glorified wrapper around that. @cw9 can chime in on the algo stuff

Reply:
this is fine, the algo basically calls the `add()` and then call the `remove()` on non sharded placements